Dockerfile

```diff
@@ -50,6 +50,7 @@ DEBIAN_FRONTEND=noninteractive apt-get install -y --allow-unauthenticated \
     python3-pip \
     python3-venv \
     rocblas \
+    rocrand \
```

Comment:
@junliume In the `ci_d868bf` docker, rocrand is included with ROCm 6.0 installation, so this line is not necessary.

Reply:
@atamazov I'm investigating it. Meanwhile I try to update CI docker to ROCm 6.1 RC2, and a few things are missing there.

Reply:
Reverted these changes, restarted CI and let's see, I recall an error reporting rocrand is missing.